README.md

```diff
@@ -114,7 +114,7 @@ load(
 )
 load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
 load(
-    "@rules_xcodeproj//xcodeproj:defs.bzl",
+    "@com_github_buildbuddy_io_rules_xcodeproj//xcodeproj:defs.bzl",
```

Comment:
This has changed. I'll mention something in the next release notes, but `@rules_xcodeproj` is now the recommended way.

Reply:
(It's odd how the README.md and the most-recent release are mismatched, but I hope to release a new release by Monday EOD.)

Reply:
I imagine most folks go and checkout the great examples anyway